rs/pocket_ic_server/CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   specifies if non-mainnet features (e.g., best-effort responses) should be enabled for the PocketIC instance.
   The topology contains a new field `subnet_seed` which is equal to the directory name of the directory in the `state_dir`
   storing the state of the corresponding subnet.
+- Support for canister HTTP outcalls: endpoint `/instances/<instance_id>/get_canister_http` to retrieve pending canister HTTP outcalls
+  and endpoint `/instances/<instance_id>/mock_canister_http_response` to mock a response for a pending canister HTTP outcall,
+  the server produces responses for pending canister HTTP outcalls automatically in the auto-progress mode (started by calling the endpoint `/instances/<instance_id>/auto_progress`).
```

Comment:
For my understanding: where does it get the responses from? does it make the actual https outcall, or is it mocked (or both)?

Reply:
> where does it get the responses from? 

in the auto progress mode, the PocketIC server actually makes the http calls to the requested URL; otherwise the test driver needs to get the response somehow (make an http call, mock the response) and call the `mock_canister_http_response` endpoint; please see the doc comment on `get_canister_http`

Reply:
Thanks. Maybe you can add more explicitly to the `get_canister_http` docstring that in live mode, the framework makes the actual call over the internet to the specified URL

Reply:
Done